db/seeds.rb

```diff
@@ -30,6 +30,21 @@ def create_if_none(klass, count = nil)
 puts "Seeding with multiplication factor: #{SEEDS_MULTIPLIER}\n\n"
 
 ##############################################################################
+# Default development site config if different from production scenario
+
+SiteConfig.public = true
+SiteConfig.waiting_on_first_user = false
+
+##############################################################################
+
+# Put forem into "starter mode"
+
+if ENV["MODE"] == "STARTER"
```

Comment:
Adds first new "mode", I imagine we could create more different scenarios this way for better developer experience.

Like for spinning up different configs, or putting us in different scenarios of follow behavior, types of listings created, chat channel situations, etc.